instrumentation/restlet/restlet-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/restlet/v1_0/ServerInstrumentation.java

```diff
@@ -71,6 +75,12 @@ public static void finishRequest(
 
       scope.close();
 
+      if (response.getStatus() != null
+          && response.getStatus().equals(Status.CLIENT_ERROR_NOT_FOUND)) {
```

Comment:
NIT: this can be easily compacted to Status.CLIENT_ERROR_NOT_FOUND.equals(response.getStatus()) 
In general comparing constant - to - value allows to skip a nullcheck :)

Reply:
Certainly more elegant, thanks!